src/app/QuickStarts/quickstarts/start-a-recording.tsx

```diff
@@ -157,40 +153,39 @@ Downloading a recording will save the recording to your local machine as a JFR f
 1. Click the [kebab menu]{{highlight recording-kebab}} next to the recording that you want to view an analysis report for.
 2. Click \`View Report ...\` to view an analysis report of the recording in a new tab.
-
 3. *Optional:* Right click on the page and select \`Save Page As...\` to download the report HTML file to your local machine.
 `,
         review: {
           instructions: '#### Verify that you can see an analysis report of the recording.',
           failedTaskHelp:
-            'The kebab icon `⁝` should be on the right end of the recording row in the active recordings table. Clicking the kebab icon should show a menu with the `View Report ...` option.',
+            'The kebab icon `⁝` should be next to the recording row in the active recordings table. Clicking the kebab icon should show a menu with the `View Report ...` option.',
         },
       },
       {
         title: 'Archive a recording',
         description: `
-Archiving a recording will save the recording to [APP]'s archival storage, and will persist even after either the target JVM, or [APP], has stopped. These recordings will show up in the target JVM's Archived Recordings tab, as well as in the [Archives]{{highlight nav-archives-tab}} view on the [APP] console navigation bar.
+Archiving a recording will save the recording to [APP]'s archival storage, and will persist even after either the target JVM, or [APP], has stopped. These recordings will appear in the target JVM's Archived Recordings tab, as well as in the [Archives]{{highlight nav-archives-tab}} view on the [APP] console navigation bar.
 
-1. Click the [Archive]{{highlight recordings-archive-btn}} button to archive the recording.
-2. Go to the [Archived Recordings]{{highlight archived-recordings-tab}} tab to see the archived recording in [APP]'s storage.
+1. Click [Archive]{{highlight recordings-archive-btn}} to archive the recording.
+2. To view the archived recording in [APP]'s storage, go to the [Archived Recordings]{{highlight archived-recordings-tab}} tab.
 
 [You can download archived recordings and view an analysis report of the archived recording from the [Archived Recordings]{{highlight archived-recordings-tab}} tab, similar to active recordings.]{{admonition tip}}`,
         review: {
           instructions: '#### Verify that the recording has been archived in the Archived Recordings tab.',
           failedTaskHelp:
-            'The recording name should have been saved like `<jvm-alias>_<recording-name>_<timestamp>.jfr`. If you still do not see the recording, try the proceeding tasks again.',
+            'The recording name should have been saved in the format <jvm-alias>_<recording-name>_<timestamp>.jfr. If you still cannot find the recording, please try the preceding steps again.',
```

Comment:
Seems like `<>` might be missed interpreted as an html tag.

![Screenshot from 2023-04-21 03-56-34](https://user-images.githubusercontent.com/68053619/233577769-12a1f334-0bec-4247-9d45-cccfa78bc52f.png)

Reply:
can we use code format here?